tests/integ/datasource/test_databricks.py

```diff
@@ -63,7 +63,7 @@ def create_databricks_connection():
 
 @pytest.mark.parametrize(
     "input_type, input_value",
-    [("table", TEST_TABLE_NAME), ("query", f"(SELECT * FROM {TEST_TABLE_NAME})")],
```

Comment:
would it break if we don't remove the ()?
if so we need to make it backward-compatible, like removing the extra parentheses by ourselves